Project.toml

```diff
@@ -9,6 +9,7 @@ CircularArrays = "7a955b69-7140-5f4e-a0ed-f168c5e2e749"
 Distances = "b4f34e82-e78d-54a5-968a-f98e89d6e8f7"
 IterTools = "c8e1da08-722c-5040-9ed9-7db0dc04731e"
 LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
+MeshViz = "9ecf9c4f-6e5a-4b5e-83ae-06f2c7a661d8"
```

Comment:
We can't add MeshViz as a dependency :)

Reply:
?? Never changed this file! Not intentional.